packages/admin/admin/src/form/Autocomplete.tsx

```diff
@@ -90,7 +89,9 @@ export const FinalFormAutocomplete = <
                     endAdornment={
                         <InputAdornment position="end">
                             {loading && <CircularProgress color="inherit" size={16} />}
-                            {clearable && <ClearInputAdornment position="end" hasClearableContent={Boolean(value)} onClick={() => onChange("")} />}
+                            {value !== null && !required && !rest.disabled && !rest.readOnly && (
```

Comment:
Can we keep this consistent, to either use only destructured values (`required`, `disabled`, ...), or only use values from the `rest` object (`rest.required`, `rest.disabled`)? 
Not just in this file, but across all components in this PR.